pkg/lib/bundle/generate.go

```diff
@@ -123,6 +126,31 @@ func GenerateFunc(directory, outputDir, packageName, channels, channelDefault st
 		return err
 	}
 
+	if synthesizeCSV {
+		if err := synthesize.Generate(directory, outputDir, overwrite); err != nil {
+			return err
+		}
+	}
+
+	if !skipValidation {
+		dir := directory
+		if outputDir != "" {
+			dir = outputDir
+		}
+
+		if results, err := synthesize.ValidateFunc(dir); err != nil {
```

Comment:
This is responding to @ecordell 's comment on when do we not want to validate a CSV. I personally agree that i want to scope this to synthesized CSV so that the default is not to examing all CSVs. So i will move this as suggested for now.